docs/teams.md

```diff
@@ -16,8 +16,8 @@ export class TeamAwesome extends ApplicationTeam {
             users: [
                 new ArnPrincipal(`arn:aws:iam::${YOUR_IAM_ACCOUNT}:user/user1`),  
                 new ArnPrincipal(`arn:aws:iam::${YOUR_IAM_ACCOUNT}:user/user2`)
-            ]
-
+            ],
+            manifestDir: './examples/teams/team-awesome/'
```

Comment:
I think this should be `teamManifestDir`?

Reply:
Correct. Will make the change.